samples/AspNetCoreApiSample/Dto/TaskPatchDto.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Withywoods.AspNetCoreApiSample.Utils;
+
+namespace Withywoods.AspNetCoreApiSample.Dto
+{
+    /// <summary>
+    /// Task Patch DTO (Data Transfer Object).
+    /// </summary>
+    public class TaskPatchDto : PropertiesChangedTracker
```

Comment:
Remove PropertiesChangedTracker inheritance (not used) and use simple get set properties